src/uu/head/src/head.rs

```diff
@@ -1,240 +1,557 @@
-//  * This file is part of the uutils coreutils package.
-//  *
-//  * (c) Alan Andrade <alan.andradec@gmail.com>
-//  *
-//  * For the full copyright and license information, please view the LICENSE
-//  * file that was distributed with this source code.
-//  *
-//  * Synced with: https://raw.github.com/avsm/src/master/usr.bin/head/head.c
-
-#[macro_use]
-extern crate uucore;
-
-use std::collections::VecDeque;
-use std::fs::File;
-use std::io::{stdin, BufRead, BufReader, Read};
-use std::path::Path;
-use std::str::from_utf8;
-
-static SYNTAX: &str = "";
-static SUMMARY: &str = "";
-static LONG_HELP: &str = "";
-
-enum FilterMode {
-    Bytes(usize),
+use clap::{App, Arg};
+use std::convert::TryFrom;
+use std::ffi::OsString;
+use std::io::{ErrorKind, Read, Seek, SeekFrom, Write};
+use uucore::{crash, executable, show_error};
+
+const EXIT_FAILURE: i32 = 1;
+const EXIT_SUCCESS: i32 = 0;
+const BUF_SIZE: usize = 65536;
+
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+const ABOUT: &str = "\
+    Print the first 10 lines of each FILE to standard output.\n\
+    With more than one FILE, precede each with a header giving the file name.\n\
+    \n\
+    With no FILE, or when FILE is -, read standard input.\n\
+    \n\
+    Mandatory arguments to long flags are mandatory for short flags too.\
+    ";
+const USAGE: &str = "head [FLAG]... [FILE]...";
+
+mod options {
+    pub const BYTES_NAME: &str = "BYTES";
+    pub const BYTES_HELP: &str = "\
```

Comment:
for consistency, please move that in the clap call

Reply:
Done (I'm assuming you're referring only to the `HELP` messages)